### DISCUSSION TITLE
Putting two pictures on top of each others renders them gray scale rather than colored

### DISCUSSION THREAD

**Discussion by Tuttu** [ImageMagick/ImageMagick#5724] [Help] [answered]
Hello,

I'm using IM to merge two pictures, one on top of the other, the top one with transparency. The top picture is a black icon and the bottom one is colored. When I merge them, the rendered pictured is in gray scale rather than staying colored and I can't find why. 
That's even more weird because I already used the same command to merge two pictures similarly with success. 
What am I doing wrong here?

Here are the pictures and the command line:
`magick Top.png Bottom.png -compose DstOver -composite Out.png`

Top picture: ![GFX_situation_approach_amenities](https://user-images.githubusercontent.com/996797/200126995-7fdf619b-7d13-4893-9ff9-e814ba77816f.png)
Bottom picture: ![GFX_situation_approach_bg_green](https://user-images.githubusercontent.com/996797/200127003-a3d57527-56b3-470b-97c0-1ce55e61c0c4.png)
Result: ![GFX_situation_approach_amenities+Green](https://user-images.githubusercontent.com/996797/200127008-c39b672f-2931-4bed-81ef-1297c1284960.png)

**Comment by fmw42** [comment, marked as answer]
Imagemagick use the type of the first image in the command for the output. Switch your input image order and use -compose over so that the color image is first.

`magick Bottom.png Top.png -compose over -composite out.png`

![Out](https://user-images.githubusercontent.com/13458218/200131445-be92fe70-93d6-4db4-a00e-cabbc15b27ba.png)

**Reply by Tuttu** [reply]
It worked like a charm, thanks. I wasn't aware about this particularity, being quite new with Imagemagick.
